wafer/talks/views.py

```diff
@@ -239,8 +241,14 @@ def get_context_data(self, **kwargs):
         context = super(Speakers, self).get_context_data(**kwargs)
         speakers = UserProfile.objects.filter(
             user__talks__status='A').distinct().prefetch_related(
-                'user').order_by('user__first_name', 'user__last_name')
-        context["speaker_rows"] = self._by_row(speakers, 4)
+            'user').order_by('user__talks__talk_type__name',
```

Comment:
I imagine we'll want to be able to select the ordering of the talk types by something other than name, but happy to leave that for a future PR.

Reply:
I've changed this to just sort on talk type (which we can control via the order attribute), so it will match the ordering on the talks list.

Reply:
Sounds good.